### DISCUSSION TITLE
Straight line & circle rulers + a symmetry tool

### DISCUSSION THREAD

**Discussion by 6xolotl** [keiwando/toonsquid#184] [Ideas]
For when you need perfect circles, perfect straight edges, or perfect symmetry.

The shape tools work for the first two in a pinch, but it’s a hassle to make all the shapes (spending a long time to get the perfect proportions) and then rasterize them. This would also allow us to make perfect circles and straight lines with the textured pixel brushes, such as the pencils, which can be useful for storyboards and animatics :]

And I think everyone knows the usefulness of a symmetry tool :3
(If it doesn’t take too much work, it would also be cool if you could move the center point for said tool, so you don’t have to draw everything in the middle of the canvas and then drag it to where it needs to be, and maybe if you could rotate the angle so you can get things just the way you need them!)

**Comment by Lioncub901** [comment]
Nice, shape ruler is great

**Reply by Lioncub901** [reply]
Also Symmetry
